crates/transport/src/copy_bidirectional.rs

```diff
@@ -0,0 +1,123 @@
+//! Fork of https://github.com/tokio-rs/tokio/blob/master/tokio/src/io/util/copy.rs to allow us set
+//! variable length `CopyBuffer` size instead of default 8k.
```

Comment:
```suggestion
//! Vendored code from https://github.com/tokio-rs/tokio/blob/master/tokio/src/io/util/copy.rs
//! It is modified to allow us setting the `CopyBuffer` size instead of hardcoding 8k.
```